components/Footer.js

```diff
@@ -0,0 +1,32 @@
+import HouseLine from "@/public/Icons/home-4-line.svg";
+import styled from "styled-components";
+import PlantFavIcon from "@/public/Icons/plant-line.svg";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <StyledLink href={"/"}>
```

Comment:
Consider adding `aria-label` attributes here for accessibility, so screen readers can interpret the purpose of each link.  Because screen readers might not interpret the purpose of an icon-only link without additional context, so here’s how you could improve it:

```suggestion
      <StyledLink href={"/"} aria-label="Go to homepage">
```